proto/qdrant/cloud/iam/v1/iam.proto

```diff
@@ -498,3 +511,15 @@ enum UserConsentStatus {
   // period of time, we auto accept pending consents depending on the document type.
   USER_CONSENT_STATUS_PENDING = 3;
 }
+
+// buf:lint:ignore QDRANT_CLOUD_REQUIRED_REQUEST_FIELDS
+// LogoutRequest is the request for the Logout function
+message LogoutRequest {
+  // Empty
+}
+
+// LogoutResponse is the response from the Logout function
+message LogoutResponse {
+  // The actual user.
+  User user = 1;
```

Comment:
Do we need to return the user after logging out? I'd prefer to return an empty response in this case if possible.